.github/workflows/build_gtfs_validator.yaml

```diff
@@ -10,13 +10,20 @@ jobs:
       - name: Code Checkout
         uses: actions/checkout@v2
 
+      - name: Auth GCP
+        id: 'auth'
+        uses: 'google-github-actions/auth@v1'
+        with:
+          credentials_json: '${{ secrets.GOOGLE_CREDENTIALS }}'
+          create_credentials_file: true
```

Comment:
looks like `create_credentials_file` [defaults to true](https://github.com/google-github-actions/auth#other-inputs), so we could omit it. but i don't feel strongly, being explicit has benefits too

Reply:
Good callout, I didn't notice that. I'd usually remove it, but I think given the subtle interaction this command has with later commands (it populates that file/env var) I think I'll keep it just to be extra explicit. While figuring out the initial error here it took me a while to remember this was how things are working, maybe it'll save us some future hassle haha